### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Tinram**
Updated the README.md to include connection instructions for the application.
On first use of the app, I expected something like innotop's connection, but had to look at the source code to find the mysql url.
Providing this in the README will help prospective users (and may be later a --help/-h switch).

### CODE REVIEW

**Comment by datacharmer** [comment]
Good idea.
Even better, in addition to the README, is having innotopgo suggest the syntax:
```diff
diff --git a/main.go b/main.go
index 7b8f1bf..80355b4 100644
--- a/main.go
+++ b/main.go
@@ -13,6 +13,7 @@ func main() {
        var displaytype = "simple"
        if len(os.Args) < 2 {
                fmt.Println("No URI provided !")
+               fmt.Println("Syntax: innotopgo mysql://<username>:<password>@<host>:3306")
                os.Exit(1)
        }
        if len(os.Args) < 3 {
```

**Comment by Tinram** [comment]
Yes, definitely better.

File: main.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 func main() {
 	var displaytype = "simple"
 	if len(os.Args) < 2 {
-		fmt.Println("No URI provided !")
+		fmt.Println("Usage: innotopgo mysql://<username>:<password>@<host>:3306")
```
Comment (lefred) [inline main.go:15]: if 3306 is used, then the port is not mandatory.
I would suggest to have something like : 

`   mysql://<username>[:<password>]@<host>[:<port>]`

**Review by lefred — CHANGES_REQUESTED** [review]
Thank you for your pull request.

I would suggest this change

File: main.go (hunk 1 above)
Comment (Tinram) [reply main.go:15]: Yes, I agree on the port text change, and have just tested it (to verify myself).
But the password is now not mandatory?  Is that assuming a bad practice blank default MySQL password?!

Comment (lefred) [reply main.go:15]: You are right, I was confused with MySQL Shell integration of the credential-store-helper  (I thought the driver would use it).
But it's indeed not the case.

**Pull request merged**